src/frontend/src/binder/expr/binary_op.rs

```diff
@@ -137,6 +137,23 @@ impl Binder {
                     }
                 }
             }
+            BinaryOperator::Custom(name) if name == "&&" => {
```

Comment:
@xiangjinwu when should we use `Custom` vs directly add the operator to the AST? I suppose when there's no english name for the operator? `&&` for instance doesn't have an english name.